tests/console/systemd_bsc_1241079.pm

```diff
@@ -0,0 +1,141 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: systemd_regression
+# Summary: https://bugzilla.suse.com/show_bug.cgi?id=1241079#c13
+# - follow steps mentioned in https://github.com/systemd/systemd/issues/10627
+# - check no systemd hang/core dump/crash
+# - test other service like 'sshd' is working fine
+# Maintainer: qe-core <qe-core@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use power_action_utils 'power_action';
+use utils;
+
+
+# script to reload systemd daemon and edit a test service
+my $script
+  = 'while :; do dbus-send --system --print-reply --dest=org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager.Reload; sleep 1; done & { systemctl edit --full _test.service; fg; }';
+
+sub install_broken_pkgs {
+    # Install broken systemd packages
+    my ($self) = @_;
+    assert_script_run('curl -o bsc_1241079.tar ' . data_url('bsc_1241079.tar'));
+    assert_script_run('tar xf bsc_1241079.tar');
+    assert_script_run('rpm -Uvh --force bsc_1241079/*.rpm');
+
+    power_action('reboot', textmode => 1);
+    $self->wait_boot;
+}
+
+sub add_test_service {
+    select_serial_terminal;
+    script_output(
+        "echo  \"\$(cat <<EOF
+[Service]
+Type=simple
+WorkingDirectory=/tmp
+ExecStart=/usr/bin/sleep 100
+EOF
+        )\"  >> /run/systemd/system/_test.service"
+    );
+}
+
+sub run_daemon_reload {
+    select_console 'root-console';
+    # When editor opens, do at least a single change, e.g. add a new empty line.
+    enter_cmd("$script");
+    wait_still_screen 3;
+    type_string 'o';
+    send_key 'ret';
+    send_key 'esc';
+    type_string ':x';
+    send_key 'ret';
+}
+
+sub run_2nd_daemon_reload {
+    select_console 'user-console';
+    enter_cmd("sudo -i", wait_still_screen => 1);
+    enter_cmd("root", wait_still_screen => 1);
+    enter_cmd("$testapi::password", wait_still_screen => 1);
+    enter_cmd("$script");
+    wait_still_screen 3;
+    type_string 'o';
+    send_key 'ret';
+    wait_still_screen 3;
+    record_info "Check screenshot to see if any issue";
```

Comment:
This test is too long, it will not show. IMO it's better to use short text/word in title and longer text in second field.

Reply:
btw it does not do any check, so it's misleading, I would just call it what it is Screenshot